APE0.rst

```diff
@@ -0,0 +1,360 @@
+involve the community whenever possible for decisions, and delegating
+responsibilities.
+
+When delegating its power by creating roles or making appointments the
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its powers, the Coordination Committee must first attempt to reach a
+consensus. If a consensus cannot be reached the Committee votes. Every
+Coordination Committee member must either vote or explicitly abstain. Members
+with a conflict of interest on a particular vote must abstain. The final
+decision regarding what constitutes a conflict of interest is determined by the
+Ombudsperson in accordance with the conflict of interest policy. Passing
+decisions require a strict majority of non-abstaining Coordination Committee
+members.
+
+Whenever possible, the Coordination Committee's deliberations shall be held in
+public in a manner that permits discussion and comments from the community.
+Final decisions made by the Coordination Committee will be published in a
+publicly visible location.
+
+Electing the Coordination Committee
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A Coordination Committee election is held annually to fill any vacant seats.
+Notice of the election must be posted via the Project’s standard communication
+channels (reaching all active members) at least one month before the election begins.
+The election consists of two phases, each lasting two weeks:
+
+* Phase 1, Nominations: Candidates will be nominated and confirm their interest
+  in serving. Candidates can be nominated by anyone, including themselves, and
+  do not have to be Voting Members to be nominated.
+* Phase 2, Voting: Each Voting Member can vote for zero or more of the
+  candidates, up to the number of candidates. Voting is performed anonymously.
+  Candidates are ranked by the total number of votes they receive. If a tie
+  occurs, it may be resolved by mutual agreement among the candidates, or else
+  the winner will be chosen at random.
+
+The election process is managed by a non-voting Returns Officer nominated by the
+outgoing Coordination Committee. For the initial election, the Returns Officer
+will be nominated by the NumFOCUS Executive officer or their delegate. Elections
+will be carried out every year, provided that at least one of the Coordination
+Committee members has reached the end of their term or decided to step down.
+
+Term
+^^^^
+Each Coordination Committee member's term runs for three years from when
+the election results are finalized, except when the election was to fill a
+vacancy created partway through a term. In that case, the newly-elected member’s
+term runs for the remainder of the term of the person they are replacing. There
+is no limit to the number of terms that a single individual can be elected for.
+
+For the initial election of Coordination Committee members, two seats will have
```

Comment:
A technical question here: In the initial vote or later on if positions with different length are to be filled (say one full term position and one 1 year replacement), how do we decide who gets which seat?

Are we running independent elections for each seat? If so, can people run for both the 3 year seat and the 1 year seat at the same time?

Suggested solution: In that case, the person with the largest number of votes gets the longest available seat.

Reply:
My memory was that the WG sort of intentionally left this vague as something to be dealt with as part of the "implementation" of the election.  But of course we can lock it down now if we think it's important!

I think I like your suggestion but with an adjustment to allow some agency: that the person with the largest number of votes gets first "pick" at the slot.  For example, I could imagine someone seeing the outcome of the election and saying "actually, I'd rather this other person get the longest-running seat because I may want to step down next year anyway".  But I could live with that *or* your suggestion, @hamogu

Reply:
How about a combo of these: The person with the largest number of votes gets the longest available seat unless the candidates mutually agree to something different.

Reply:
Sounds good to me.